lib/Options.pm

```diff
@@ -92,7 +94,9 @@ my %cap_defaults = (
 			   quiet => 0,
 			   days => 5,
 			   offset => 0,
+			   timeout => 240,			   
 			   output => undef,
+			   agent => 'Mozilla/5.0 (Linux) XmlTv',			   
```

Comment:
@ejonesnospam interesting choice of the UA string. What if someone is using XMLTV on Windows, Mac or some other non-Linux platform? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) the UA string is: `<product> / <product-version> <comment>`.

May I suggest simply making it `Mozilla/5.0 XMLTV`? (Feel free to ignore my suggestion, as I am in no way affiliated with this project :smile: )